atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/SweepResults.java

```diff
@@ -29,9 +32,23 @@
 @Value.Immutable
 public abstract class SweepResults {
 
-    public abstract Optional<byte[]> getPreviousStartRow();
+    @Nullable
+    @JsonProperty("previousStartRow")
+    protected abstract byte[] getPreviousStartRowNullable();
+
+    @Nullable
+    @JsonProperty("nextStartRow")
+    protected abstract byte[] getNextStartRowNullable();
+
+    @JsonIgnore
```

Comment:
I don't think this would work if we persisted these anywhere and needed to deserialize them, but it looks like we don't.

Reply:
why not?

Reply:
Nevermind, missed the new JsonProperty keys